Automattic/newspack-newsletters
# Add branding banner to the CPT wp-admin pages

### All Submissions:






### Changes proposed in this Pull Request:

Very simple Newspack banner added to the CPT page using CSS-only as I didn't want to tweak the PHP of the pages.

![Screenshot 2020-08-27 at 17 16 33](https://user-images.githubusercontent.com/177929/91469074-d4b53e00-e88a-11ea-84c7-afcaa937a7cb.png)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Do you see the blue banner when visiting the different sections of the plugin (All Newsletters, Settings, Ads)
3. Try to install the plugin for the first time (no Settings). Does it clash with our dismissible banner?

### Other information:
